src/SLCore.Listeners/SLCore.Listeners.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ConnectedMode\ConnectedMode.csproj" />
+    <ProjectReference Include="..\Integration\Integration.csproj" />
```

Comment:
I think it is a bad idea to add a dependency from SLCore.Listeners assembly to the Integration assembly, which according to this diagram (https://lucid.app/lucidchart/6905ec0c-fb12-4fc5-b0f3-37ca9cf3c997/edit?invitationId=inv_513583e6-c1e2-4cc9-9051-2f54f09c845e&page=0_0#) is used for VS integration. 

I think instead IUnsupportedNodeVersionNotificationService can be moved inside Core, then we can remove this dependency.

Reply:
That's the whole purpose of the separate Listeners assembly, to allow us to reference these kind of assemblies. ConnectedMode.csproj already has a dependency to that.

Reply:
As discussed, please give it a try to move the IUnsupportedNodeVersionNotificationService to a better place, that will allow us to remove the dependency to Integration. If it's not easy, we can discuss again and I can approve the PR.